lib/rules/require-render-return.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       recommended: true,
       url: docsUrl('require-render-return')
     },
-    schema: [{}]
+    schema: []
```

Comment:
this seems like it'd be a breaking change, in case someone had already been providing an empty object.

Reply:
it technically would be breaking in the strictest sense.
Though reviewing the history, this has been an empty object for 3 years, so I doubt there's some legacy config people have left over.

The problem is that this schema value doesn't just allow an empty object, it allows literally value.

Reply:
yes - i agree that's a problem, however, that means that removing this would cause eslint to crash on a config that was previously working :-/